modules/setting/setting.go

```diff
@@ -1344,6 +1345,7 @@ func newMailService() {
 
 		UseSendmail:  sec.Key("USE_SENDMAIL").MustBool(),
 		SendmailPath: sec.Key("SENDMAIL_PATH").MustString("sendmail"),
+		SendmailArgs: sec.Key("SENDMAIL_ARGS").Strings(" "),
```

Comment:
@cez81 this does not handle quoted or escaped strings I think.

Reply:
@geek1011 New commits to use go-shellquote library